### PR TITLE
Arithmetic Operations

### PR DESCRIPTION
**Pull request by Deepalakshmi20**

### CODE REVIEW

File: Deepalakshmi.B/ArithmeticOperations Ruby/cal_controller.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  def div
+  	@num1=params[:n1].to_i
+  	@num2=params[:n2].to_i
+  	@div= @num1 / @num2
+  	render json: {message: "Division of #{@num1} by #{@num2} is #{@div}"}
+  end
+
+  private
+   def validation
+   	if !(params[:n1].present? && params[:n2].present?)
```
Comment (InbasasiS) [inline Deepalakshmi.B/ArithmeticOperations Ruby/cal_controller.rb:34]: instead !, params[:n1].blank? can be used

File: Deepalakshmi.B/ArithmeticOperations Ruby/cal_controller.rb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      render json: {  message: "check your inputs" } 
+    end 
+
+  end
+
+  def div_by_zero
+  	if params[:n2].to_i==0
+    render json: { message: "Denominator can't be zero"}
```
Comment (InbasasiS) [inline Deepalakshmi.B/ArithmeticOperations Ruby/cal_controller.rb:42]: render json: { status: 400, message: "check your inputs" } 

Use 400 status code incase of bad inputs.

**Comment by InbasasiS** [comment]
Good Work @Deepalakshmi20 
Just note down those comments.
Merging it 
cc: @AkbarHabeeb @tamilarasumk731 @smartsri18

File: Deepalakshmi.B/ArithmeticOperations Ruby/cal_controller.rb (hunk 1 above)
Comment (Deepalakshmi20) [reply Deepalakshmi.B/ArithmeticOperations Ruby/cal_controller.rb:34]: Okay anna.

**Pull request merged**